comment/migrations/0009_auto_20200811_1945.py

```diff
@@ -7,8 +7,9 @@
 def set_default_email(apps, schema_editor):
     comment_model = apps.get_model('comment', 'Comment')
     for comment in comment_model.objects.all():
-        comment.email = comment.user.email
-        comment.save(update_fields=['email'])
+        if getattr(comment.user, 'email', False):
```

Comment:
This can probably be replaced with
```suggestion
        if hasattr(comment.user, 'email'):
```